tests/Aspire.Hosting.Tests/MySql/AddMySqlTests.cs

```diff
@@ -2,10 +2,10 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 
 using System.Net.Sockets;
-using Aspire.Hosting.MySql;
 using System.Text.RegularExpressions;
 using Microsoft.Extensions.DependencyInjection;
 using Xunit;
+using Aspire.Hosting.MySql;
 using Aspire.Hosting.Utils;
 using Aspire.Hosting.Tests.Utils;
```

Comment:
Nitpick: based on other files (and general .NET convention) `using Aspire` should come before `using Microsoft` (but after `using System`) due to alphabetical ordering.